src/libs/environment/parseBooleanVariables.test.ts

```diff
@@ -20,4 +21,7 @@ describe('parseBooleanVariables', () => {
   it('should not touch strings other than "true" and "false"', () => {
     expect(convertedConfig.WEBSOCKET_ENDPOINT).toEqual('websocket-endpoint')
   })
+  it('should not touch strings other than "true" and "false"', () => {
+    expect(convertedConfig.SENTRY_DSN).toEqual('sentry-dsn')
+  })
 })
```

Comment:
Je pense que c'est ok de mettre tous les `expect` dans le même `it`.
Ou alors de séparer en deux tests pour les variables boolean et les variables string.
Qu'en penses-tu ?

Reply:
j'avais pas remarqué cette segmentation. plutôt en 3 je dirai: falsy, truthy and string

Reply:
yes